v3/generate.go

```diff
@@ -73,11 +73,23 @@ func modifyDefinitions(version string, isClient bool, name string, def map[inter
 
 	switch name {
 	case "Student":
+		if version == "v3.0" {
+			delete(properties, "home_language_name")
+			delete(properties, "home_language_code")
+		}
 		if !isClient {
 			delete(properties, "iep_status")
 			delete(properties, "home_language")
 			delete(properties, "unweighted_gpa")
 			delete(properties, "weighted_gpa")
+		} else {
+			if version > "v3.0" {
+				delete(properties, "home_language")
```

Comment:
Might want to add a comment in code here noting that home_language in v3.1 onwards is split into home_language_name and home_language_code